frontend/src/component/common/SegmentItem/SegmentItem.tsx

```diff
@@ -48,12 +48,13 @@ const StyledAccordionDetails = styled(AccordionDetails)(({ theme }) => ({
     padding: theme.spacing(0.5, 3, 2.5),
 }));
 
-const StyledLink = styled(Link)({
+const StyledLink = styled(Link)(({ theme }) => ({
     textDecoration: 'none',
+    paddingRight: theme.spacing(0.5),
```

Comment:
This is probably totally fine, but why are you adding padding to the link? Doesn't look like it's got a different background color, so it'll just be invisible? If this is about spacing, isn't it better to add it to the relevant parent element?